content/chronograf/v1.8/guides/annotations.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Using annotations in Chronograf views
-description: Annotations provide contextual information by adding explanatory notes or comments to Chronograf graph views and charts in the user interface and dashboards.
+title: Use annotations in Chronograf views
+description: Add contextual information to Chronograf dashboards with annotations.
 menu:
   chronograf_1_8:
-    name: Using annotations
+    name: Use annotations
     weight: 50
     parent: Guides
 ---
 
-## Using annotations in the Chronograf interface
+## Use annotations in the Chronograf interface
```

Comment:
I don't know if we even need this h2. It basically just repeats the h1 that will appear above it.